policies/omc-bylaws.html

```diff
@@ -152,7 +152,7 @@ <h3><a name="OMC">OpenSSL Management Committee (OMC)</a></h3>
           to vote on and participate in discussions. They retain access to OMC
           internal resources.</p>
 
-          <h4><a name="voting">OMC Voting Procedures</a></h4>
+          <h4><a name="omc-voting">OMC Voting Procedures</a></h4>
```

Comment:
I don't believe that this constitutes a change to the bylaws even though it touches the file.

Reply:
Oh, but I just noticed that I failed to change the other "voting" tag to "otc-voting", which was the main reason why I did it in the first place:

https://github.com/openssl/web/blob/43d51d911e5110ac0115d5929720f684c01a2d2b/policies/omc-bylaws.html#L265

Reply:
Fixed in aaac5205218b291075557ef1af2cdf238566761c.